pkg/build/api/v1/types.go

```diff
@@ -658,6 +672,15 @@ type BuildRequest struct {
 
 	// Env contains additional environment variables you want to pass into a builder container
 	Env []kapi.EnvVar `json:"env,omitempty" description:"additional environment variables you want to pass into a builder container"`
+
+	// BuildTriggerReason
+	BuildTriggerReason BuildTriggerReason `json:"env,omitempty" description:"a reason why a build was triggered"`
+}
+
+type BuildTriggerReason struct {
+	User           string `json:"env,omitempty" description:"user"`
```

Comment:
@PI-Victor you need different names for each field, try to change this to `json:"user"`, and the same for the fields below: `triggerType`, `triggerImageID`. HTH